### ISSUE TITLE
< causes insertion of messages with indentation

### ISSUE THREAD

**Issue by shlandturtle** [HDInnovations/UNIT3D#469]
### Issue Type
- [x] Bug Report
- [ ] Feature Idea
- [ ] Enhancement
- [ ] Technical Discussion
- [ ] Question

### End User Info
-- UNIT3D Version: #.#.# unknown

-- Laravel Version: #.#.# unknown

-- PHP Version: unknown

-- Database Driver & Version: unknown

-- Web Server Driver & Version: unknown

-- OS Driver and Version: unknown

### Expected Behavior:
< shouldn't cause insertion of the resulting message nor indention to the messages.

### Actual Behavior:
When < is used in chat without other characters, next posted chat message gets inserted into this message. This can go on and on for many messages, effectively having empty messages and indention on the messages.

### Steps To Reproduce The Problem:
1. < and then enter
2. post another post into the chatbox

### Possible Fix:

**Comment by JackPriceBurns** [comment]
In `App\Http\Controllers\API\ChatController.php` line `71`... add this follow if check as a possible fix?

```php
        if ($message == '<') {
            return response('error', 401);
        }
```

**Comment by HDVinnie** [comment]
kinda hackish @JackPriceBurns , we will just fix the HTMLPurifier used.

**Comment by JackPriceBurns** [comment]
Yeah I felt it was hacky as well but I couldn't seem to find where this HTMLPurifier was used haha

**Comment by HDVinnie** [comment]
@JackPriceBurns https://github.com/mewebstudio/Purifier in case your curious
